### PR TITLE
G0 4715 | Prod- On the delete popup msg was not proper the changes need to implement from every pages

### PR DESCRIPTION
**Pull request by ravinderwalkover**
G0 4715 | Prod- On the delete popup msg was not proper the changes need to implement from every pages

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 1]
```diff
@@ -1276,7 +1276,7 @@ <h3 class="modal-title bg" id="modal-title">Confirmation </h3>
                 <div class="modal-body pdL2 pdR2 clearfix" id="export-body">
                     <form name="newRole" novalidate class="" autocomplete="off">
                         <div class="modal_wrap mrB2">
-                            <h3>Do you want to delete the selected entries? </h3>
+                            <h3>Do you want to delete the selected entrie(s)? </h3>
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/ledger.component.html]: Should be entry(s)

File: apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts [hunk 2]
```diff
@@ -2,21 +2,22 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
...
 
-	public onConfirmation() {
-		this.confirmDeleteEvent.emit(true);
-	}
-
-	public onCancel() {
-		this.closeModelEvent.emit(true);
-	}
+    public onCancel() {
+        console.log(this.module);
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts]: Remove consoles

File: apps/web-giddh/src/app/shared/helpers/confirmationMessages.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export interface ConfirmationMessages {
+    module: string;
+    title: string;
+    message1: string;
+    message2: string;
+}
+
+export const confirmationMessages: ConfirmationMessages[] = [
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/helpers/confirmationMessages.ts]: Add JS Doc to interface and constant

File: apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts [hunk 4]
```diff
@@ -2,21 +2,22 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
...
 
 @Component({
-	selector: 'delete-role-confirmation-model',
-	templateUrl: './confirmation.model.component.html'
+    selector: 'delete-role-confirmation-model',
+    templateUrl: './confirmation.model.component.html'
 })
 
 export class DeleteRoleConfirmationModelComponent {
+    @Input() public module: string = '';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts]: Add JS doc to newly added input

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 1 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html]: Please check the card and check the message mentioned there.

File: apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts (hunk 4 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts]: Check now

File: apps/web-giddh/src/app/shared/helpers/confirmationMessages.ts (hunk 3 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/helpers/confirmationMessages.ts]: Check now

File: apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts (hunk 2 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.ts]: Removed

**Pull request merged**